app/routes.py

```diff
@@ -1,2 +1,54 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, abort, make_response
 
+class Planet:
+    def __init__(self, id, name, moons, description):
+        self.id = id
+        self.name = name
+        self.moons = moons
+        self.description = description 
+
+
+planets = [
+    Planet(1, "Mercury", 0, "The first planet."),
+    Planet(2, "Venus", 0, "The second planet."),
+    Planet(3, "Earth", 1, "The third planet.")
+
+]
+
+planets_bp = Blueprint("planets", __name__, url_prefix = "/planets")
+
+
+@planets_bp.route("", methods=["GET"])
+def handle_planets():
+    planets_response = []
+    for planet in planets:
+        planets_response.append({
+            "id": planet.id,
+            "name": planet.name,
+            "moons": planet.moons,
+            "description": planet.description
+        })
```

Comment:
this could be moved to your planets class as an instance method. Then you would call that instance method here.
```python
 def make_dict(self):
        return dict(
            id=self.id,
            name=self.name,
            moons=self.moons,
            description=self.description,
        )
```